README.md

```diff
@@ -20,17 +20,19 @@ The following general libs and utilities are installed:
 
 The build tools included are:
 
-- **Java** openjdk 11.0.2 2019-01-15
```

Comment:
Why change to latest?

Would soon be wrong

Reply:
Latest is the wrong word I agree. I just mean that the versions arent pinned (will update whenever the image is rebuilt)

Reply:
I was going to update all the versions here but I know they'd become out of date before long

Reply:
I've now removed the "latest" wording entirely for unpinned stuff

Reply:
that seems like the wrong thing to be doing you don't want your tools changing from one ci build to the next.
 
Should fix the base version then they would be pinned.

The tests could be extended to print all these version to make it easier to update.